plugins/pixart-tp/pixart-tp.quirk

```diff
@@ -0,0 +1,17 @@
+[HIDRAW\VEN_093A&DEV_4F07]
+Plugin = pixart_tp
+GType = FuPxiTpDevice
+HidVersionReg = bank=0x00;addr=0xb2
+SramSelect = 0x0f
```

Comment:
this needs to have a PxiTp prefix -- e.g. make it `PxiTpSramSelect` -- it also has to be declared in `fu_pxi_tp_plugin_constructed()` with `fu_context_add_quirk_key()` too